modules/ROOT/pages/6.7.2-release-notes.adoc

```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.
```

Comment:
It's the integrator or developer who calls the function and that causes an error. I don't think we even use that function that is why we missed it not being executed with the right context.

```suggestion
The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when an integrator called the 'getModifierState' function on a **keydown** event action.
```

Reply:
The method `getModeierState` is called on an event object,  not an event action.